### ISSUE TITLE
Docs mention "mvel" as default scripting language, should be "painless"

### ISSUE THREAD

**Issue by themowski** [olivere/elastic#1054]
While looking up how to do scripted updates, I noticed that the documentation for the `*Script.Lang()` function in `elastic/script.go` is outdated/incorrect. It states that the default is `"mvel"`, which (from what I gather) is the old/original default scripting language -- the current default (6.x) is `"painless"`.

I'm not sure if this is purely a documentation fix, or if you're also setting that value somewhere in code -- I haven't had time yet to experiment with the functionality.

### Which version of Elastic are you using?

[x] elastic.v6 (for Elasticsearch 6.x)
[ ] elastic.v5 (for Elasticsearch 5.x)
[ ] elastic.v3 (for Elasticsearch 2.x)
[ ] elastic.v2 (for Elasticsearch 1.x)

### Please describe the expected behavior

* The documentation for the `*Script.Lang()` function should indicate that the default is `"painless"`. 

* The URL provided in the docstring also maps to the Elasticsearch 6.2 docs. I'm not sure if you want to update that to the latest 6.x release, or even just use current? https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-scripting.html

* I don't know if `"mvel"` is even used anymore? If it's not (the official docs don't seem to indicate that it's a valid option), then it should be removed.

* Are `"groovy"` and `"python"` actually supported? They're not on the official list, but they are in the docstring for `*Script.Lang()`.

### Please describe the actual behavior

From `elastic/script.go#L66`:

```go
// Lang sets the language of the script. Permitted values are "groovy",
// "expression", "mustache", "mvel" (default), "javascript", "python".
// To use certain languages, you need to configure your server and/or
// add plugins. See https://www.elastic.co/guide/en/elasticsearch/reference/6.2/modules-scripting.html
// for details.
func (s *Script) Lang(lang string) *Script {
	s.lang = lang
	return s
}
```

This also shows up on the docs website: https://godoc.org/github.com/olivere/elastic#Script.Lang

### Any steps to reproduce the behavior?

N/A

**Comment by olivere** [comment]
Yes. These types of things are hard to track. Will fix.

I'm not sure, but I guess Groovy and Python are deprecated or—at least—cannot be found in the official repository. Maybe they exist as separate plugins. So it should be possible to specify them in the uncertain case that somebody uses custom plugins for scripting.

**Comment by themowski** [comment]
> Yes. **These types of things are hard to track.** Will fix.

Agreed, arguably the hardest part of any project is keeping up with documentation, but in situations where your docs depend on other peoples' docs... 😵  You've done a fantastic job so far keeping things well-documented and up to date.

I did some digging (which should have been done before I logged this ticket, sorry) and found that Elasticsearch 6.0 removed support for "python", "groovy", and "javascript" (they were all deprecated in Elasticsearch 5.0):

https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-6.0.html#breaking_60_scripting_changes

It looks like there's also a new "java" option that allows custom plugin-based scripting.

In light of that, how about this as a suggested updated docstring? (I bumped the version in the URL to 6.6, but we could use 6.0 or leave it as 6.2. I'm hesitant to use current because that would point to 7.x when it comes out.)

```go
// Lang sets the language of the script. Permitted values are
// "painless" (the default if otherwise unspecified), "expression",
// "java", and "mustache". See
// https://www.elastic.co/guide/en/elasticsearch/reference/6.6/modules-scripting.html
// for additional details.
```
